src/pages/detail/PensionDetailPage.jsx

```
@@ -77,230 +78,88 @@ const PensionDetailPage = () => {
         return;
       }
 
-      const headers = {
-        Accept: 'application/json',
-        ...(accessToken && { Authorization: `Bearer ${accessToken}` }),
-      };
-
       await axios.post(
```

ESLint

[error] 'axios' is not defined.

(no-undef)